src/Model/Modules/Library.php

```diff
@@ -1117,6 +1117,17 @@ public function getChangelogPath(Version $version)
         return $version->injectPattern($pattern);
     }
 
+    public function getChangelogTemplatePath()
+    {
+        $cowData = $this->getCowData();
```

Comment:
With PHP 7.1 now a requirement, you could do `return $this->getCowData()['changelog-template'] ?? null;` right?